### PR TITLE
Add default PR reviewers, remind to add project -- closes #75

### LINKED ISSUE #75: Update pull requests' defaults for reviewers & project
**Do not disclose security vulnerabilities here, please email us instead: 
az-digital-security@list.arizona.edu**

## Conditions of satisfaction
- [ ] A submitted pull request should include az-digital/developers as reviewers
- [ ] A submitted pull request should be added to az quickstart project

### PR DESCRIPTION
**Pull request by barrettbaffert**
## Description

- [ ] A submitted pull request should include az-digital/developers as reviewers
- [ ] Add a checklist item to remind folks to include the az quickstart project

unfortunately github doesn't allow a default project for a pull request template

## Related Issue

https://github.com/az-digital/az_quickstart/issues/75

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I've added this Pull Request to the AZ Quickstart project in the right column.
```
Comment (trackleft) [inline .github/pull_request_template.md:26]: We should link to the project from here

Comment (trackleft) [reply .github/pull_request_template.md:26]: We should also link to the CONTRIBUTING.md document below.

Comment (trackleft) [reply .github/pull_request_template.md:26]: [AZ Quickstart project](https://github.com/orgs/az-digital/projects/1)
 [CONTRIBUTING](https://github.com/az-digital/az_quickstart/blob/master/CONTRIBUTING.md)
``` [AZ Quickstart project](https://github.com/orgs/az-digital/projects/1)```
 ```[CONTRIBUTING](https://github.com/az-digital/az_quickstart/blob/master/CONTRIBUTING.md)```

Comment (joeparsons) [reply .github/pull_request_template.md:26]: This will be reconsidered to also account for ZenHub.

**Pull request merged**